tbx/static_src/sass/components/_event-block.scss

```diff
@@ -82,20 +83,42 @@
         color: var(--color--text);
     }
 
+    &__meta-item-value {
+        font-weight: $weight--semibold;
+    }
+
     &__divider {
-        margin: $spacer-medium 0 $spacer-small;
+        margin: $spacer-small 0;
         color: rgba($color--white, 0.2);
+
+        @include media-query(large) {
+            margin: $spacer-medium 0 $spacer-small;
+        }
     }
 
     &__link {
         @include link-styles($color--white, $color--white);
         width: fit-content;
     }
 
-    &__picture {
+    &__picture-wrapper {
         height: 430px;
+        position: relative;
+        order: 1;
+
+        @include media-query(large) {
+            order: 2;
+            height: 100%;
+        }
+    }
+
+    &__picture {
         @include media-query(large) {
-            height: 500px;
+            position: absolute;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
```

Comment:
[picky] could use `inset: 0` instead of the top/left/right/bottom

Reply:
good point, I always forget about inset